src/app/pages/camera-view/camera-view.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CameraViewComponent } from './camera-view.component';
+
+describe('CameraViewComponent', () => {
+  let component: CameraViewComponent;
+  let fixture: ComponentFixture<CameraViewComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ CameraViewComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CameraViewComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
won't need the testing files for final project

Reply:
Angular has a cli command that'll make components for you using "ng generate component" and the testing file comes with it.